docroot/sites/all/modules/features/ilr_experience_reports/ilr_experience_reports.module

```diff
@@ -7,18 +7,12 @@
 include_once 'ilr_experience_reports.features.inc';
 
 /**
- * Implements hook_init().
+ * Implements hook_node_view().
  */
-function ilr_experience_reports_init() {
-  global $theme;
-  if ($theme == 'ilr_theme') {
-    drupal_add_js(drupal_get_path('module', 'ilr_experience_reports') . '/ilr_experience_reports.js', array(
-        'type' => 'file',
-        'scope' => 'footer'
-      ));
-
-    $inview_js = libraries_get_path('inview') . '/jquery.inview.min.js';
-    drupal_add_js($inview_js, array('type' => 'file', 'scope' => 'footer'));
+function ilr_experience_reports_node_view($node, $view_mode, $langcode) {
+  $selector = 'node-type-experience-report';
```

Comment:
Can you please move the `$selector` line in this and in ilr_visiting_fellows.module inside the `if` statement?